frame/pablo/src/lib.rs

```diff
@@ -284,10 +294,22 @@ pub mod pallet {
 
 		/// Required origin for pool creation.
 		type PoolCreationOrigin: EnsureOrigin<Self::Origin>;
+
+		/// Time provider.
+		type Time: Time;
+
+		/// The interval between TWAP computations.
+		#[pallet::constant]
+		type TWAPInterval: Get<MomentOf<Self>>;
+
+		/// Provider that tell up whether a specific pool TWAP is enabled or not.
+		type EnabledPoolTWAP: Contains<(CurrencyPair<Self::AssetId>, Self::PoolId)>;
```

Comment:
So this means we'd need to have a third pallet with extrinsics for adding to the twaps?